technical-reports/format/types.md

```diff
@@ -12,21 +12,42 @@ If an explicit type is set, but the value does not match the expected syntax the
 
 ## Color
 
-Represents a 24bit RGB or 24+8bit RGBA color in the sRGB color space. The `$type` property MUST be set to the string `color`. The `$value` property MUST be a string containing a [hex triplet/quartet](https://www.w3.org/TR/css-color-4/#typedef-hex-color) including the preceding `#` character. To support other color spaces, such as HSL, translation tools SHOULD convert color tokens to the equivalent value as needed.
+Represents a color in the UI. The `$type` property MUST be set to the string `color`. The `$value` property MUST be an object string containing a string `colorSpace`, a numeric `coordinates` array, optional numeric `alpha`, string `colorProfile`, and string `fallback` to support legacy color spaces as needed.
+
+| Key            |      Type      | Required | Description                                                                                                                              |
+| :------------- | :------------: | :------: | :--------------------------------------------------------------------------------------------------------------------------------------- |
+| `colorSpace`   |    `string`    |    Y     | An string representing the color space.                                                                                                  |
+| `colorProfile` |    `string`    |          | String representing url of color profile.                                                                                                |
+| `coordinates`  | `number array` |    Y     | An array of numeric values representing individual color coordinates.                                                                    |
+| `alpha`        |    `number`    |          | An integer or floating-point value representing the numeric value.                                                                       |
```

Comment:
We should specify this MUST be in the range of `(0, 1)` (inclusive). `0` = 0%, `1` = 100%.

Reply:
Can do. Where should this sentence occur?

Reply:
Anywhere in the `alpha` description